app/preprints/-components/submit/title-and-abstract/component.ts

```diff
@@ -17,24 +19,35 @@ interface TitleAndAbstractForm {
     description: string;
 }
 
-const TitleAndAbstractFormValidation: ValidationObject<TitleAndAbstractForm> = {
-    title: validatePresence({
-        presence: true,
-        ignoreBlank: true,
-        type: 'empty',
-    }),
-    description: validatePresence({
-        presence: true,
-        ignoreBlank: true,
-        type: 'empty',
-    }),
-};
-
 /**
  * The Title And Abstract Component
  */
 export default class TitleAndAbstract extends Component<TitleAndAbstractArgs>{
-    titleAndAbstractFormChangeset = buildChangeset(this.args.manager.preprint, TitleAndAbstractFormValidation);
+    @service intl!: Intl;
+    titleAndAbstractFormValidation: ValidationObject<TitleAndAbstractForm> = {
+        title: validatePresence({
+            presence: true,
+            ignoreBlank: true,
+            type: 'empty',
+        }),
+        description: [
+            validatePresence({
+                presence: true,
+                ignoreBlank: true,
+                type: 'empty',
+            }),
+            validateLength({
+                min: 30,
```

Comment:
According to https://openscience.atlassian.net/browse/ENG-5717 this should be 20